### ISSUE TITLE
StreamsConnection.get_domains() raises exception

### ISSUE THREAD

**Issue by ibmkendrick** [IBMStreams/streamsx.topology#912]
```
sc = rest.StreamingAnalyticsConnection('vcap_services.json', 'Streaming-Analytics')
domains = sc.get_domains()
```
result in:
```
Traceback (most recent call last):
  File "test2.py", line 31, in <module>
    domains = sc.get_domains()
  File "/home/streamsadmin/Documents/git/streamsx.topology/com.ibm.streamsx.topology/opt/python/packages/streamsx/rest.py", line 92, in get_domains
    return self._get_elements('domains')
TypeError: _get_elements() missing 1 required positional argument: 'eclass'
```

**Comment by ibmkendrick** [comment]
`sc.get_instances()` is also not working, as it just returns an empty array, even when the instance is running.

**Comment by ibmkendrick** [comment]
I'll fix this as part of the `StreamsConnection` documentation effort.

**Comment by ddebrunner** [comment]
Please try to keep the general doc updates separate from bug fixes.

**Comment by ibmkendrick** [comment]
Fixed by PR #926
